pylib/cqlshlib/cql3handling.py

```diff
@@ -1443,7 +1443,7 @@ def alter_type_field_completer(ctxt, cass):
                               ( "SUPERUSER" | "NOSUPERUSER" )?
                         ;
 
-<alterUserStatement> ::= "ALTER" "USER" <username>
+<alterUserStatement> ::= "ALTER" "USER" ("IF" "EXISTS")? <username>
```

Comment:
Despite the original intent of the ticket mentioning only `ALTER` if we're going to touch `ALTER USER` maybe we should also do `DROP USER` and friends, wdyt @ekaterinadimitrova2 @blerer ? Otherwise it feels like this is only half way through to me.